icons/citigroup.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
Some of the issues mentioned in the [linter](https://github.com/simple-icons/simple-icons/actions/runs/8032783903/job/21942557759?pr=10496) can be fixed using SVGO. Take a look a the section "Optimize the Icon" of our [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon).

Reply:
After that, you'll be able to follow the "Annotate the Icon" sections of our [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon).